### PR TITLE
New rule `selector-pseudo-class-no-unknown`.

### PR DESCRIPTION
**Pull request by evilebottnawi**
And fixed stupid bug

### CODE REVIEW

**Comment by jeddy3** [comment]
@evilebottnawi Can you pull that bug fix into a separate PR please. So I can include it in `6.3.3.`.

**Comment by evilebottnawi** [comment]
@jeddy3 yes, working

**Comment by evilebottnawi** [comment]
@davidtheclark @jeddy3 ping

File: src/rules/selector-pseudo-class-no-unknown/__tests__/index.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+  }, {
+    code: "a:pseudo-class { }",
+    message: messages.rejected(":pseudo-class"),
+    line: 1,
+    column: 2,
+  }, {
+    code: "a:unknown::before { }",
+    message: messages.rejected(":unknown"),
+    line: 1,
+    column: 2,
```
Comment (davidtheclark) [inline src/rules/selector-pseudo-class-no-unknown/__tests__/index.js]: I think it's worth adding a test where the error should appear in a different location to verify that the location is working property, e.g. `a,\nb > .foo:error { }`

File: src/utils/isKnownPseudoClass.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+/**
+ * Check is known pseudo-class
+ *
+ * @param {string} Pseudo-class
+ * @return {boolean} If `true`, the unit is known
+ */
+
+const knownPseudoClasses = new Set([
```
Comment (davidtheclark) [inline src/utils/isKnownPseudoClass.js]: Can you link to where you got this information?

**Comment by davidtheclark** [comment]
LGTM, just the two comments.

**Comment by evilebottnawi** [comment]
@davidtheclark ping

**Comment by davidtheclark** [comment]
Great. Go ahead and merge after resolving conflicts.

**Comment by evilebottnawi** [comment]
Green CI. Merge.

**Pull request merged**